packages/components/molecules/f-media-element/package.json

```diff
@@ -54,8 +54,6 @@
   },
   "devDependencies": {
     "@vue/cli-plugin-babel": "4.5.16",
```

Comment:
Is this one still needed?

Reply:
Yep, was getting issues with the babel optional chaining without it. Will be looking at a fix for this in a future ticket 👍🏼